cmd/chunk/main.go

```diff
@@ -77,6 +78,13 @@ func init() {
 	rootCmd.Flags().Int64VarP(&chunkSize, "chunk-size", "s", chunk.DefaultChunkSize, "maximum size of each HTTP request done using the content range header.")
 	rootCmd.Flags().IntVarP(&concurrencyPerServer, "concurrency-per-server", "c", chunk.DefaultConcurrencyPerServer, "controls the max number of concurrent connections opened to the same server.")
 	rootCmd.Flags().BoolVarP(&restartDownloads, "force-restart", "f", chunk.DefaultRestartDownload, "restart previous downloads, ignoring where they were stopped")
+	rootCmd.Flags().StringVarP(
+		&progressDir,
+		"progress-directory",
```

Comment:
Given the `CHUNK_DIR`, I would suggest calling this flag only `--dir`. It is neat! :D

Reply:
I think it's better to keep `--progress-directory` because in that context the _main_ directory in the mind of the user might be the `--output-directory` (around line 75). What do you think?

Reply:
Good point. SGTM.